docs/data-model.rst

```diff
@@ -96,6 +97,7 @@ Bytecode is represented by an object that contains:
 * An unlinked bytes array (``bytes``)
 * A list of any known link references (``linkReferences``)
 * Possibly a mapping to known source ranges (``sourceMap``)
+* An instruction object containing information derived from the bytecode and sourcemap (``instruction``)
```

Comment:
This should say something like `An array of instruction objects containing...`, and have the corresponding correct field `instructions`